src/styles/mainClassDefinitions.less

```diff
@@ -294,6 +293,10 @@
 		user-select: auto;
 		float: left;
 
+		&:focus {
+			color: #fff;
```

Comment:
It occurs to me that this isn't quite right.  We need to distinguish not only when the input has focus, but also need to know when the radio button is enabled/disabled.  Working on it now.